zeebe/atomix/cluster/src/main/java/io/atomix/raft/cluster/impl/RaftMemberContext.java

```diff
@@ -151,10 +148,7 @@ private void resetReaderAtEndOfLog(final RaftLogReader reader) {
   public boolean canAppend() {
     return open
         && (inFlightAppendCount == 0
-            || (appendSucceeded
-                && inFlightAppendCount < maxAppendsPerMember
-                && System.currentTimeMillis() - (timeStats.getMean() / maxAppendsPerMember)
-                    >= appendTime));
```

Comment:
❓ So it means before we haven't sent an append if it was slow? SO this means we consequently preferred some followers, right? I think this is what we also sometimes in benchmarks 🤔 

Is the consequence clear enough for us? Is it fine that tests are running fine? Might be worth to also run some benchmark for it. In general, I agree that we have several of these statistics/heuristics here and there, which we probably don't need.

Reply:
Can we then also remove the `appendTime`?

Reply:
Yes, I agree that we should also run a benchmark before merging this as the consequences can be a bit hard to estimate.

To be honest I don't really understand why we should wait for `meanAppendLatency/maxAppends` since the last appendStart before issuing a new append (in the case there are is an inflight "slot" available)

Reply:
When discussing it with @npepinpe, he proposed that long-term we should look into implementing something with AIMD #30379

Reply:
Atomix legacy 🫨 🤷🏼 


I tracked it back to:


 * https://github.com/zeebe-io/atomix/commit/c784afc19670b0b1d2d98f01fd2033582336b245
 * https://github.com/zeebe-io/atomix/commit/7965e8f73eea4334e9170983356d9a0d18dc0f8b
 * https://github.com/zeebe-io/atomix/commit/9e9623b0d1d7ff51e9fc3d1c424b08ec79f62e60

I have no clear reason for it